src/analysis/exponential.lean

```diff
@@ -320,6 +329,18 @@ lemma cos_eq_one_iff (x : ℝ) : cos x = 1 ↔ ∃ n : ℤ, (n : ℝ) * (2 * π)
         exact absurd h (by norm_num))⟩,
   λ ⟨n, hn⟩, hn ▸ cos_int_mul_two_pi _⟩
 
+theorem cos_eq_zero_iff (θ : ℝ) : cos θ = 0 ↔ ∃ k : ℤ, θ = (2 * k + 1) * pi / 2 :=
+begin

```

Comment:
theta should be implicit

Reply:
Wait, do you mean implicit or explicit? I think it should be explicit...

Reply:
Usually with `iffs` I prefer implicit because they usually end up being used as `cos_eq_zero_iff.1`.